bin/config-abbrev

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
+var CONFIG_FILENAME = 'deploy-config.js';
+var DEFAULT_CONFIG_ABBREV = 7;
+
+var path = require('path');
+var getConfigFor = require('../tasks/utils').getConfigFor;
+
+var config = getConfigFor('git', true);
+var abbrev = (config && config.abbrev) || DEFAULT_CONFIG_ABBREV;
+
+process.stdout.write(abbrev.toString());
```

Comment:
The `bin/` folder is meant for executables. I would suggest to move this to `utils.js`, `tasks/` or some new folder.